### DISCUSSION TITLE
[Art competition] Dos Attack

### DISCUSSION THREAD

**Discussion by snurf198** [prosyslab-classroom/cs348-information-security#398] [Art competition]
![image](https://github.com/prosyslab-classroom/cs348-information-security/assets/55132820/49b18c51-a98b-4655-8adf-89178c7c59ae)

## Short Description
For me, "Dos Attack" means "aggressive girl challenging with other girls who are trying to take over a attractive man" 

## Long Description (up to 3 sentences)
Dos Attack sends numerous requests to prevent the computer from processing actual requests, making the service unavailable. Likewise, in order to take over a attractive man, an active woman will keep making appointments with the man so that no one but herself can get over him.

**Comment by KihongHeo** [comment]
Where is the attractive man? (yourself?)

BTW, it is a coincidence that I see this picture while watching "I am solo".
